src/UI/templates/js/Input/Field/tagInput.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Wraps the BootstrapTagsInput
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+var il = il || {};
+il.UI = il.UI || {};
+il.UI.Input = il.UI.Input || {};
+(function ($) {
+    il.UI.Input.tagInput = (function ($) {
+
+
+        /**
+         *
+         * @param raw_id
+         * @param config
+         */
+        var init = function (raw_id, config) {
+            var _DEBUG = false;
+            var _CONFIG = {
+                id: '',
+                options: [],
+                selected_options: [],
+                extendable: false,
+                suggestion_starts: 1,
+                max_chars: 2000,
+                suggestion_limit: 50,
+                debug: _DEBUG,
+                allow_duplicates: false,
+                highlight: true,
+                hint: true,
+                tag_class: "label label-primary"
+            };
+
+            var _ELEMENTS = {
+                hidden_template: null,
+                container: null
+            };
+
+            var _log = function (key, data) {
+                if (!_DEBUG) {
+                    return;
+                }
+                console.log("***********************");
+                console.log(key + ":");
+                console.log(data);
+            };
```

Comment:
Hate this to be places here.  We need a more generic logging for js, like il.log, which is then injected into some component. However, not today.